predicators/nsrt_learning/strips_learning/gen_to_spec_learner.py

```diff
@@ -45,6 +45,133 @@ def _create_general_pnad_for_option(
 
         return pnad
 
```

Comment:
would you mind pointing out if any of the code below is changed?

Reply:
Nothing changed!